src/sentry/workflow_engine/models/data_condition_group.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, region_silo_model, sane_repr
+
+from .detector import Detector
+from .workflow_action import WorkflowAction
+
+
+@region_silo_model
+class DataConditionGroup(DefaultFieldsModel):
+    """
+    A data condition is a way to specify a condition that must be met for a workflow to execute.
```

Comment:
`A data group is a way to specify a group of conditions that must be met for a workflow action to execute`